hkdf.go

```diff
@@ -214,6 +226,31 @@ func ExpandHKDFOneShot(h func() hash.Hash, pseudorandomKey, info []byte, keyLeng
 	return out, nil
 }
 
+// ExpandTLS13KDF derives a key from the given hash, key, label and context. It will use
+// "TLS13-KDF" algorithm to do so.
+func ExpandTLS13KDF(h func() hash.Hash, pseudorandomKey, label, context []byte, keyLength int) ([]byte, error) {
+	if !SupportsTLS13KDF() {
+		return nil, errUnsupportedVersion()
+	}
+
+	md, err := hashFuncToMD(h)
+	if err != nil {
+		return nil, err
+	}
+
+	out := make([]byte, keyLength)
+
+	ctx, err := newTLS13KDFExpandCtx3(md, []byte(label), context, pseudorandomKey)
```

Comment:
`label` is already a `[]byte`.

```suggestion
	ctx, err := newTLS13KDFExpandCtx3(md, label, context, pseudorandomKey)
```

Reply:
thanks.